core/controlplane/config/config.go

```diff
@@ -779,6 +808,37 @@ type SelfHosting struct {
 	TyphaImage      model.Image `yaml:"typhaImage"`
 }
 
+type CoreOS struct {
```

Comment:
`HostOS` or `ContainerLinux`? I may prefer the former because the functionality isn't specific to Container Linux.

Reply:
Can we move this into the `model` pkg for better organization?